features/support/journey_helpers.rb

```diff
@@ -41,13 +41,26 @@ def old_select_lower_tier_options
 end
 
 def select_random_upper_tier_options(carrier)
+  return answer_random_upper_tier_questions(carrier) unless @resource_object == :new_registration
+
+  i = rand(2)
+  if i == 0
```

Comment:
I would rather "Help me decide" was the more common option here so that we can test the flows more frequently.

Consider if i != 0?

Reply:
Well, the `rand` function is pretty trustable on giving a random number in the range with an equal frequency for all available numbers. The only difference is that using `rand(2)` is more obvious how many option we have there (2, for instance) but the count will start from `0` so the possibilities will be `0` or `1`. Changing the  if to be `if i !=0` won't make any change to the code run

Reply:
Ah, OK. I mistakenly assumed rand(2) gave options of 0, 1 or 2.